testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListActivity.kt

```diff
@@ -1,21 +1,36 @@
 package com.rakuten.tech.mobile.testapp.ui.miniapplist
 
+import android.os.Build
 import android.os.Bundle
+import android.text.Html
+import android.text.Spanned
+import android.view.View
+import androidx.core.text.HtmlCompat.FROM_HTML_MODE_LEGACY
 import com.rakuten.tech.mobile.miniapp.testapp.R
+import com.rakuten.tech.mobile.testapp.helper.MiniAppListHelper
 import com.rakuten.tech.mobile.testapp.ui.settings.AppSettings
 import com.rakuten.tech.mobile.testapp.ui.settings.SettingsMenuBaseActivity
+import kotlinx.android.synthetic.main.mini_app_list_activity.*
 
 class MiniAppListActivity : SettingsMenuBaseActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        MiniAppListHelper.instance.getMiniAppListFromStorage()
```

Comment:
This isn't being set to a variable, so looks like this isn't being used for anything?

Reply:
oops sorry for this silly mistake.